.travis.yml

```diff
@@ -2,11 +2,11 @@ sudo: false
 language: ruby
 cache: bundler
 rvm:
-  - 2.1
-  - 2.2
+  - 2.3.4
```

Comment:
We should add Ruby `2.2` here as well. Use `2.3` and `2.4`. (See `dkdeploy-core`)